### PR TITLE
Added UITextView text's producer

### PR DESCRIPTION
**Pull request by RuiAAPeres**

### CODE REVIEW

File: Source/UIKit/UITextView.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import UIKit
+import enum Result.NoError
+
+extension UITextView {
+    
+    /// Sends the textView's string value whenever it changes.
+    public var rex_textSignal: SignalProducer<String, NoError> {
+        return NSNotificationCenter.defaultCenter()
+            .rac_notifications(UITextViewTextDidChangeNotification, object: self)
+            .filterMap { notification in
```
Comment (Rupert-RR) [inline Source/UIKit/UITextView.swift]: Just a suggestion (not necessarily any better, depending on how you value code clarity etc), but might something like `.filterMap { ($0 as? UITextView)?.text }` be simpler?

Comment (RuiAAPeres) [inline Source/UIKit/UITextView.swift]: That is not equivalent. the `$0` is a `NSNotification`. `$0.object` is a `UITextView`. I am ok with the change:

```
    public var rex_textSignal: SignalProducer<String, NoError> {
        return NSNotificationCenter.defaultCenter()
            .rac_notifications(UITextViewTextDidChangeNotification, object: self)
            .filterMap  { ($0.object as? UITextView)?.text }
    }
```

But then I would like to change the `UITextField` (for consistency sake). @neilpa what do you think?

Comment (Rupert-RR) [inline Source/UIKit/UITextView.swift]: Sorry, yes — that was just me not paying enough attention.
`($0.notification.object as? UITextView)?.text`

Comment (neilpa) [inline Source/UIKit/UITextView.swift]: If you want to make that change and update `UITextField` I'm fine with that.

**Pull request merged**